applications/serial_lte_modem/doc/SOCKET_AT_commands.rst

```diff
@@ -1269,6 +1275,191 @@ Test command
 
 The test command is not supported.
 
+Asynchronous socket polling #XAPOLL
+===================================
+
+The ``#XAPOLL`` command allows you to receive URC notifications for events on all opened sockets or for selected sockets that have already been opened.
+
+Set command
+-----------
+
+The set command allows you to activate or deactivate asynchronous polling for sockets.
+
+Syntax
+~~~~~~
+
+::
+
+   #XAPOLL=<op>,<events>[,<handle1>[,<handle2> ...<handle8>]
+
+* The ``<op>`` value can accept one of the following values:
+  ``0`` stop asynchronous polling.
+  ``1`` start asynchronous polling.
+
+* The ``<events>`` value is an integer.
+  It represents the events to poll for, which can be a combination of ``ZSOCK_POLLIN`` and ``ZSOCK_POLLOUT``.
+  Permanent error and closure events (``ZSOCK_POLLERR``, ``ZSOCK_POLLHUP`` and ``ZSOCK_POLLNVAL``) are polled by default.
+  The value can be set to one of the following:
+  ``0`` Poll only the default events.
+  ``1`` Read events (``ZSOCK_POLLIN``) are polled, in addition to the default events.
+  ``4`` Write events (``ZSOCK_POLLOUT``) are polled, in addition to the default events.
+  ``5`` Read and write events are polled, in addition to the default events.
```

Comment:
```suggestion
* The ``<events>`` value is an integer.
  It represents the events to poll for, which can be a combination of ``POLLIN`` and ``POLLOUT``.
  Permanent error and closure events (``POLLERR``, ``POLLHUP`` and ``POLLNVAL``) are polled by default.
  The value can be set to one of the following:
  ``0`` Poll only the default events.
  ``1`` Read events (``POLLIN``) are polled, in addition to the default events.
  ``4`` Write events (``POLLOUT``) are polled, in addition to the default events.
  ``5`` Read and write events are polled, in addition to the default events.
```

Don't use the `ZSOCK_` prefix.
I don't think it adds any value here.
Some users don't care that it is Zephyr based product.